docs/api-ref/prefect/cli/project.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Prefect command line interface for working with Prefect projects.
```

Comment:
Do we also want to rename this file?

Reply:
I didn't deprecate the CLI `project` module, just the `project` command group within it. Do you think that we need to?

Reply:
I think we should leave this file in and update the module doc string for `prefect.cli.projects` to communicate that it's deprecated and point to the preferred commands. That way there's a path for the user to follow in the docs.

Reply:
Yeah, I agree, this commit should do that [74c840e](https://github.com/PrefectHQ/prefect/pull/9944/commits/74c840ea7889a78bcf4beb82a0fc13520f484ab4)